demo/index.html

```diff
@@ -26,18 +26,20 @@
   <link rel="import" href="../../iron-icons/communication-icons.html">
   <link rel="import" href="../paper-badge.html">
   <link rel="import" href="test-button.html">
-
-  <style is="custom-style" include="demo-pages-shared-styles">
-    .vertical-section-container {
-      max-width: 550px;
-    }
-    paper-icon-button, test-button {
-      padding: 3px;
-      border-radius: 3px;
-      margin-left: 30px;
-      margin-right: 30px;
-    }
-  </style>
+  
+  <custom-style include="demo-pages-shared-styles">
+    <style is="custom-style" include="demo-pages-shared-styles">
```

Comment:
I think the `custom-style` doesn't need the include, it's fine just on the `style`